libs/js-sdk/src/client.ts

```diff
@@ -175,17 +186,17 @@ export function createBrowserClient({
             debugLog('updateAudiences: invoke initialised callback with false')
             initialisedState.initialisedCallbacks.forEach((c) => c(false))
 
-            state.audiences = updatedAudiences
+            stateStore.audiences = updatedAudiences
             debugLog(
                 'updateAudiences: Audiences updated (%o), getting new effective values',
                 updatedAudiences,
             )
 
-            updateStrategyImplementation.connect(state).then(() => {
+            updateStrategyImplementation.connect(stateStore).then(() => {
```

Comment:
`connect()` here can throw? We probably need to `catch` the error here and log it.